src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/profiler/ProfilerEditingTarget.java

```diff
@@ -500,8 +531,13 @@ public void onError(ServerError error)
    
    private void persistHtmlPath(String htmlPath)
    {
+      persistDocumentProperty("htmlPath", htmlPath);
```

Comment:
Yes. we store html files under `/profiler_cache`, similar to `/viewer_Cache` for the `View()` feature.